packages/near-lake-primitives/README.md

```diff
@@ -0,0 +1,356 @@
+# NEAR Lake Primitive Types
+
+This crate contains the primitive types used by the [NEAR Lake Framework package](https://www.npmjs.com/package/@near-lake/framework). These types are used to define the data structures used by the framework as well as provide some popular helper functions.
+
+## Types
+
+### `Block`
+
+#### `Block` Structure Definition
+
+The `Block` type is used to represent a block in the NEAR Lake Framework. It is comprised by the following structure:
+
+```ts
+export class Block {
+    constructor(
+        readonly streamerMessage: StreamerMessage,
+        private executedReceipts: Receipt[],
+        readonly postponedReceipts: Receipt[],
+        readonly transactions: Transaction[],
+        private _actions: Map<string, Action>,
+        private _events: Map<string, Event[]>,
+        private _stateChanges: StateChange[]) { 
+
+        }
+    ... // helper methods and getters ommitted for brevity
+}
+```
+
+##### `streamerMessage`
+
+Low-level structure for backward compatibility. As implemented in previous versions of [`near-lake-framework`](https://www.npmjs.com/package/near-lake-framework). 
+
+##### `executedReceipts`
```

Comment:
I am not sure we want to describe the private fields at all. 

The reasoning behind privatizing those fields in the first place is to forbid the developers to use them. This allows us to change the internal structure of the class while keeping the APIs (methods) the same instead of introducing the breaking changes.

So I would suggest a rule of thumb: if the field is private do not describe it in the doc.

Objections? Other thoughts?

Reply:
Makes sense to me